GitUI/Translation/English.xlf

```diff
@@ -3201,6 +3201,10 @@ gitex.cmd / gitex (located in the same folder as GitExtensions.exe):</source>
         <source>Commit</source>
         <target />
       </trans-unit>
+      <trans-unit id="AllowEmpty.Text">
+        <source>Allow &amp;Empty</source>
```

Comment:
```suggestion
        <source>Allow &amp;empty</source>
```
Could you also align the `&Amend commit` checkbox to our capitalization convention, please?